pkg/odo/cli/component/push.go

```diff
@@ -47,21 +47,27 @@ const PushRecommendedCommandName = "push"
 
 // PushOptions encapsulates options that push command uses
 type PushOptions struct {
```

Comment:
I find it somewhat weird that it's called `PushOptions`. If you see, it's being used at various places across the code. I feel that `CommonOption` would be a better name for it. Also, doesn't make sense to be put in `push.go` file, IMO.

Reply:
I think it really represents the Push Options. It is used from other commands with a `--now` flag, that calls the Push method at some point.